core_lib/src/structure/filemanager.cpp

```diff
@@ -400,6 +337,28 @@ Status FileManager::save(const Object* object, QString sFileName)
     return Status::OK;
 }
 
+Status FileManager::flush(const Object* object)
+{
+    DebugDetails dd;
+
+    QStringList filesWritten;
+
+    const QString dataFolder = object->dataDir();
+    const QString mainXml = object->mainXMLFile();
+
+    Status stKeyFrames = flushKeyFrameFiles(object, dataFolder, filesWritten);
```

Comment:
Same with this, couldn't we use writeKeyFrameFiles?

Reply:
Probably because it's dumping a lot of files rather than one...a vivid description just like flushing a toilet with a lot of water :p